port/unix/omrsysinfo.c

```diff
@@ -43,10 +43,12 @@
 #include <mach-o/dyld.h>
 #include <sys/param.h>
 #include <sys/mount.h>
+#endif /* defined(LINUX) && !defined(OMRZTPF) */
+
+#if defined(OSX) || defined(OMR_BSD)
 #include <sys/types.h>
 #include <sys/sysctl.h>
-
-#endif /* defined(LINUX) && !defined(OMRZTPF) */
+#endif() /* defined(OSX) || defined(OMR_BSD) */
```

Comment:
```suggestion
#endif /* defined(OSX) || defined(OMR_BSD) */
```

Reply:
ugh. I think I've spent too much time in cmake  😄